batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -0,0 +1,147 @@
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by romit on 1/14/16.
+ */
+@WebServlet(name = "EmployeeController", urlPatterns = {"/employees/*"})
+public class EmployeeController extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String pathInfo = request.getPathInfo();
+
+        if (pathInfo == null || pathInfo.equals("/")) {
+            list(request, response, 1);
+        } else {
+            String[] pathParts = pathInfo.split("/");
+            if (pathParts[1].equals("create")) {
+                createProcess(request, response);
+            } else if (pathParts[2].equals("delete")) {
+                int id = Integer.parseInt(pathParts[1]);
+                deleteProcess(request, response, id);
+            } else if (pathParts[2].equals("edit")) {
+                int id = Integer.parseInt(pathParts[1]);
+                editProcess(request, response, id);
+            }
+        }
+    }
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String pathInfo = request.getPathInfo();
+
+        if (pathInfo == null || pathInfo.equals("/")) {
+            list(request, response, 1);
+        } else {
+            String[] pathParts = pathInfo.split("/");
+            if (pathParts[1].equals("create")) {
+                create(request, response);
+            } else if (pathParts[2].equals("edit")) {
+                int id = Integer.parseInt(pathParts[1]);
+                edit(request, response, id);
+            }
+        }
+    }
+
+    private void list(HttpServletRequest request, HttpServletResponse response, int page) throws ServletException, IOException {
+        try {
+            EmployeeService employeeService = new EmployeeService();
+            List<Employee> empList = employeeService.fetch();
+            request.setAttribute("employeeList", empList);
+            request.getRequestDispatcher("/WEB-INF/views/employees.jsp").forward(request, response);
+        } catch (DataException e) {
+            e.printStackTrace();
+            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+            request.setAttribute("message", e.getMessage());
+            request.getRequestDispatcher("/WEB-INF/views/error.jsp").forward(request, response);
+        }
+    }
+
+    private void create(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        request.getRequestDispatcher("/WEB-INF/views/form.jsp").forward(request, response);
```

Comment:
views/form.jsp? seriously?
shouldn't this file named employeeCreate.jsp? or employee/create.jsp?